oeps/processes/oep-0055-project-maintainers.rst

```diff
@@ -0,0 +1,345 @@
+  core contributor status and maintainer status may be conferred at
+  the same time.
+
+Bootstrapping
+-------------
+In order to ensure an orderly roll-out of the formal maintainer role,
+existing 2U staff in the owner role will transition into the role of
+maintainer for the components that they own.
+
+Community members who are acting in informal maintainer roles, will
+also be asked to become named maintainers.
+
+A list of all components without maintainers will be created and
+published.
+
+An asynchronous maintainer scrum-of-scrums will be created as a Slack
+channel.
+
+Standards for measuring the quality of repositories will be rolled out
+across all repositories.  These standards can benefit from work done
+at edX to measure `repository health <https://github.com/openedx/edx-repo-health>`_.
+
+Becoming a Maintainer
+---------------------
+After the bootstrapping phase, the Open edX project will need
+processes for adding additional maintainers, replacing maintainers,
+and ensuring that new components are maintained.
+
+**This section should be congruent with the Core Contributor process**
+
+A new maintainer can be added in any of the following ways:
+
+* An existing maintainer can request that a new person become a
+  component maintainer either in conjunction with the existing
+  maintainer or replacing them.  If there is no objection from
+  existing maintainers, the role will be conferred.
+* In the case of vacancies, the nominations will be made to the
+  scrum-of-scrums to fill the vacancy.  If there is not objection from
+  existing maintainers, the role will be conferred.
+* If there consensus cannot be reached,
+* In the case of new repositories, there should be a designated
+  maintainer named before the repository is created.
+
+Removing a Maintainer
+---------------------
+From time-to-time it will be necessary to remove a component
+maintainer.  This might happen because:
+
+* A maintainer leaves the project or changes employment
+* A maintainer does not have the time or interest to continue in the
+  role
+* A maintainer is unreachable
+* A maintainer is not fulfilling the responsibilities of the role
+
+Trigger events for removing a maintainer would be:
+
+* A maintainer informs the project that the can no longer continue in
+  the role
+* A maintainer has been unreachable for more than 30 days
+* A maintainer has violated the project `Code of Conduct`_
+* A maintainer hasn't met project quality standards consistently
+  despite feedback
+
+In the case that a maintainer has violated the `Code of Conduct`_, the
+Goverance Team will decide what the repercussions of the violation
+will be, including whether the violator can continue in the role of
+maintainer.  It should be noted that maintainers are expected to be
+exemplars and, as such, will be held to a high standard.
+
+In the case that a maintainer is not meeting project quality standards
+consistently, the maintaginer will be warned with the expectation that
```

Comment:
`maintaginer` -> `maintainer`

Reply:
Who determines whether `a maintainer is not meeting project quality standards consistently` and warns/notifies the maintainer?

Reply:
@doctoryes typo fixed.  I've clarified that the SOS decides, but I think this notion needs to be further clarified throughout the document.